odata-mock/test/test.js

```diff
@@ -49,5 +49,5 @@ describe('test app', function () {
 
             done()
         }).catch(error => done(error))
-    });
+    }).timeout(50000);

```

Comment:
WHy that? that is a bit too heavy I guess. Do you had timeouts at all?

Reply:
I had on my local computer, not in pipeline. I tried with 3, still timeout, so moved it to 5.

Reply:
It is 50 seconds... For me it runs in less than 4 seconds. Are you sure it teminated at all for you

Reply:
please answer